frontend/apps/reader/modules/readertoc.lua

```diff
@@ -1203,7 +1203,7 @@ Enabling this option will restrict display to the chapter titles of progress bar
         end
     }
     menu_items.toc_items_with_dots = {
-        text = _("With dots"),
+        text = _("Tab leaders"),
```

Comment:
Not clearer, feels like technical/typographic jargon.

Reply:
that is the name of those dots, "leaders"

Reply:
These are not tabs though; this is not a document in a word processor :)

How about `Dot leaders`?

Reply:
just because they are not user adjustable, does not make them "not tabs". the ToC has a left tab for name of chapters and a right tab for page numbers.

Reply:
`tab leaders` suggests we are joining the stuff on the left tab to the stuff on the right tab (the case here)

`dot leaders` suggests the leaders use dots (instead of dashes for example)

`show leaders` could be another option.

Let's [emoji] vote then:
👍 for tab
😃 for dot
🚀 for show

Cast your votes 🗳️

Reply:
The thing is that I don't even call the chapter title and the page number on the right "tabs".
I guess you mean "tab" as web browsers' tabs.
In the context of KOReader, we nowhere use "tab" - except internally to mean tabulation, and for me, "tab" means tabulation.

As for "leaders", I only know the meaning as president/king/boss. I don't even know how this can be translated in French.
In any case, I'd want somethig with "dot" that I can guess it's about ... dots.
My vote is to keep "With dots".

Reply:
And what's the vote for no change?

Reply:
https://www.w3.org/Style/Examples/007/leaders.en.html

Reply:
> The question is: What do our readers understand more? Leaders might be the most advanced vocabulary here (with the danger that a lot of our readers wont understand it). On the other hand 'with dots' is self explaining.

I am so happy it was you who said that, i can now say the same thing you did last time, can you back that up? Bring factual data to the discussion.

my facts are, the thing is actually called “leaders” whether you like it or not is another matter. I don’t like to assume that anyone is an idiot therefore I don’t try to talk to them like one. This is kind of like saying, let’s not call dogs anymore, instead let’s use “the animal that goes woof” 

> As for "leaders", I only know the meaning as president/king/boss. I don't even know how this can be translated in French.

@poire-z i find it very difficult to respond to this sort of comments because they are essentially anecdotal evidence. I can’t guess which words someone might or might not know.

don’t forget to vote ppl.

Reply:
> i find it very difficult to respond to this sort of comments because they are essentially anecdotal evidence.

You don't need to respond anything, just record the fact that a few of our users (like me) won't get "tab leaders" or "complications". If you use "dot leaders", it's a bit better, because I'll get the word "dot" and can assume it will show something with "dots" (and I'll also consider KOReader authors pretentious snobs :))
(And I'd be ready to go with "Tab leaders" for a while, just to see what translators did with it without any @ translators explanation.)

> I can’t guess which words someone might or might not know.

Of course you can't, you're native English. That's why I (and the others who don't know it) am telling you, so you know that some words (like "complication") are not obvious to some people.

>  I don’t like to assume that anyone is an idiot

Thank you. But when you are shown enough evidence, whether you like it or not, you have to admit the fact: I'm an idiot :)

> let’s not call dogs anymore dogs, instead let’s use 

Let's not call dogs anymore hounds or mutts, let's use dogs.

> "Chefs d’onglet" according to google translate, not sure if that makes sense though
> 
> "Les points de conduite" according to the link hius07 provided

None of these are right, never read or heard these, these are auto translations - or the kind of things our translators may come up with (I've seen a few as odd as these in the French translation.)
("With dots" is translated to "Avec trainée de points", which I also never heard of, but it's quite poetic :) "with trail of dots". I also have no idea what would be the correct French terms for that.)

Reply:
The results are in, 2 votes for dot, 1 vote for tab. Congratulations to the winner constituents.

Reply:
You forgot the last minute contender :) ![image](https://github.com/koreader/koreader/assets/24273478/c9827c1a-239f-475b-98e7-f1c3e80b0dcc)

But not fighting, I'm fine bringing in the Chiefs of the Dots.

Reply:
Nah man, this was the ballot:

<img width="178" alt="image" src="https://github.com/koreader/koreader/assets/97603719/804153c1-3025-4fe9-9e99-c34a2b3d2dac">

Reply:
Btw, @Commodore64user: remember when you corrected me, saying "Show chapter lengths" was wrong, as it should be singular? How come "Dot leaders" (Or "Tab leaders", if you prefer) is acceptable here. Same principle – one dot leader per chapter, and one length per chapter. So why not "Dot leader" in this case?

Reply:
Because it is a plural only noun, like trousers or clothes. There is no such thing as one leader, two leaders. You either have leaders or you don’t. What would even be “one leader”?

Reply:
>"show chapter lenght" refers to a collective not just to a singular one

I get that, that's why I didn't argue the point any further and accepted the singular form. I still think plural would have been the better choice, however, because it immediately shows that the setting affects each ToC entry. But it's all good.